From fa3c2e3e3596daeab59a13844ff0b6f7831c70fc Mon Sep 17 00:00:00 2001
From: vmallika <vmallika@redhat.com>
Date: Fri, 4 Dec 2015 12:36:22 +0530
Subject: [PATCH 169/178] quota: add 'quota-version' to xlator volume_options structure

This is a backport of http://review.gluster.org/#/c/12878/

Below error is seen in brick log:
0-vol1-quota: option 'quota-version' is not recognized

So add this option to the volume_options structure of
marker xlator

> Change-Id: I1bd68a537936ee0f9af85dac74f1377f84edb8f4
> BUG: 1283178
> Signed-off-by: vmallika <vmallika@redhat.com>

Change-Id: I807d29a0625da0082708ed924cba7e71ba3558b5
BUG: 1287980
Signed-off-by: vmallika <vmallika@redhat.com>
Reviewed-on: https://code.engineering.redhat.com/gerrit/63353
Reviewed-by: Raghavendra Gowdappa <rgowdapp@redhat.com>
Tested-by: Raghavendra Gowdappa <rgowdapp@redhat.com>
---
 xlators/features/marker/src/marker.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/xlators/features/marker/src/marker.c b/xlators/features/marker/src/marker.c
index 5e491f6..de5b623 100644
--- a/xlators/features/marker/src/marker.c
+++ b/xlators/features/marker/src/marker.c
@@ -3385,5 +3385,6 @@ struct volume_options options[] = {
         {.key = {"inode-quota"} },
         {.key = {"xtime"}},
         {.key = {"gsync-force-xtime"}},
+        {.key = {"quota-version"} },
         {.key = {NULL}}
 };
-- 
1.7.1

